src/Command/Source/SourceReindexCommand.php

```diff
@@ -70,29 +72,34 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             if (0 > $vendorId) {
                 $output->writeln('<error>Missing vendor id required in combination with identifier</error>');
 
-                return 1;
+                return Command::FAILURE;
             }
         }
 
+        if (!is_null($lastIndexedDate) && 0 === $limit) {
+            $output->writeln('<error>"last-indexed-date" can not be given without "limit"</error>');
```

Comment:
Why? It's the other way around.... i have used it more than once without using limit